contentcuration/contentcuration/test_settings.py

```diff
@@ -0,0 +1,7 @@
+import logging
+import os
+import tempfile
+
+DEBUG = True
```

Comment:
If the only purpose of `test_settings` is to set `DEBUG` to True, this may seem redundant, as DEBUG is already true on `local_settings` (which we import after this statemtn).

Reply:
Maybe let's rename `local_settings` to `test_settings` instead, if it's primarily going to be used in there?

Reply:
Yeah, I'm actually not sure why I chose that approach. ;-) I went ahead and made the change!